test/e2e/tests/errors.spec.js

```diff
@@ -5,7 +5,8 @@ const FixtureBuilder = require('../fixture-builder');
 describe('Sentry errors', function () {
   async function mockSentry(mockServer) {
     return await mockServer
-      .forPost('https://sentry.io/api/0000000/store/')
+      .forPost('https://sentry.io/api/0000000/envelope/')
+      .withBodyIncluding('Test Error')
```

Comment:
There are multiple requests sent to the `/envelope` endpoint, but only one is for the error event. This ensures that the function call only resolves once a request is made that includes the expected error data in the body.